tox.ini

```diff
@@ -1,2 +1,12 @@
+[tox]
+envlist = py32, py33, py34
```

Comment:
@o11c We can add other Python implementations eventually. Here I just wanted to fix the problem with 3.2 and establish a way to test typing against various versions. Supporting 2.7 is the next thing I am going to do.